Vagrantfile

```diff
@@ -138,16 +144,16 @@ if File.exist?(env_specific_config_path)
 end
 
 # Display the main current configuration parameters
-@ui.info "Welcome to Kubernetes playground!"
-@ui.info "Vagrant provider: " + settings["conf"]["vagrant_provider"]
+output_info_if_enabled "Welcome to Kubernetes playground!"
```

Comment:
not a great concern, but a better and shorter name could be
`ui_info_if_enabled`
or simply
`ui_info`

Reply:
Ok for ui_info_if_enabled